
lidar2camera: 
  [ 6.7349208790990045e-02, 1.1111005592377499e-01,
       9.9152339334371908e-01, 2.0901929543910513e-01,
       -9.9769006940523020e-01, 1.6331008581773165e-02,
       6.5938028245384694e-02, -0.15,
       -8.8661990398770651e-03, -9.9367391715356890e-01,
       1.1195327991295134e-01, 6.6289995751726533e-02, 0., 0., 0., 1. ]
use_tf: false